controllers/v1/forms.js

```diff
@@ -0,0 +1,236 @@
+/**
+ * name : forms.js
+ * author : Vishnu
+ * created-date : 07-May-2024
+ * Description :  Forms Controller.
+ */
+
+// dependencies
+let formsHelpers = require(MODULES_BASE_PATH + '/forms/helper')
+
+/**
+ * Forms service.
+ * @class
+ */
+
+module.exports = class Forms extends Abstract {
+	constructor() {
+		super('forms')
+	}
+	static get name() {
```

Comment:
@Prajwal17Tunerlabs 
add validation for functions in this controller
refer : https://github.com/ELEVATE-Project/mentoring/blob/develop/src/validators/v1/form.js

Reply:
added validator